oauth2/session.go

```diff
@@ -192,3 +193,15 @@ func (s *Session) UnmarshalJSON(original []byte) (err error) {
 
 	return nil
 }
+
+func (s *Session) GetBrowserFlowCompleted() bool {
+	if s == nil {
```

Comment:
how u do u get here?this is basically instantiating a nil pointer and calling a method on it ...not sure it s realistic

Reply:
discussed on call, depends on usage, see line https://github.com/canonical/hydra/blob/16b6708f5ca838c70b7f7a219e86292c3b1fd9e2/oauth2/session.go#L127